src/components/notes/offline-indicator.styl

```diff
@@ -0,0 +1,26 @@
+.grid
+    padding 1rem
+    font-size 1rem
+    .banner &
+        font-size 0.875rem
+        line-height 1.3
+        padding 0.25rem 1rem 0.25rem 1rem
+        margin 0
+        width 100%
+
+.chip
+    background-color var(--silver)
+    opacity 60%
+
+.tooltip
+    background-color var(--paleGrey) !important
+    border-radius 8px !important
+    box-shadow 0 1px 3px 0 rgba(50, 54, 63, 0.19), 0 6px 18px 0 rgba(50, 54, 63, 0.19)
```

Comment:
should we use variables for colors?

Reply:
We need to apply some alpha transparency so we can't use our standard variables. We could set a special variable for this but shadows colors don't depends on the theme colors anyways.

Reply:
I think it is a good idea to use a variable for shadows : https://github.com/cozy/cozy-ui/blob/master/stylus/settings/shadows.styl

Reply:
J'ai du passer en CSS-in-JS pour ça, et du coup je casse la possibilité d'utiliser stylus